packages/record-data/package.json

```diff
@@ -48,8 +50,13 @@
     "ember-source-channel-url": "^2.0.1",
     "ember-try": "^1.4.0",
     "loader.js": "^4.7.0",
+    "qunit": "^2.10.0",
+    "qunit-dom": "^1.2.0",
     "silent-error": "^1.1.1"
   },
+  "ember": {
+    "edition": "octane"
+  },
```

Comment:
it's safe to octanify this as we're only adding new tests to this package at the moment and we are only a few weeks away from 4.0. Saves us a little deprecation spew.